### PR TITLE
Lookup tables - SLO

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

New SLO Lookup table feature

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203861820507288

### CODE REVIEW

File: docs/observability/reliability-management-slo/create-slo.md [hunk 1]
```diff
@@ -342,3 +342,21 @@ _view=sumologic_slo_output sloId="<your-SLO-ID>"
...
+
+You would need to create a custom graphic that combines multiple SLOs from multiple services:
+
+1. Go to **Manage Data** > **Monitoring** > **SLO**.
+1. Click on any SLO line item.
+1. Hover over the **Percentage budget remaining** panel, then click the three-dot icon > **Open in Log Search**.<br/><img src={useBaseUrl('img/observability/open-in-logsearch.png')} alt="open-in-logsearch" width="150"/>
+1. In the search field, enter the following snippet. This will join data from multiple sources for your lookup table.
+  ```
+  ??
+  ```
```
Comment (kimsauce) [inline docs/observability/reliability-management-slo/create-slo.md:361]: Need an example here

File: docs/observability/reliability-management-slo/create-slo.md [hunk 2]
```diff
@@ -342,3 +342,21 @@ _view=sumologic_slo_output sloId="<your-SLO-ID>"
 ```
 
 These log messages will be delayed by one hour, as the system ensures consistency to account for ingest delay of source telemetry.
+
+
+### Create an SLO Lookup Table
+
+If you have an existing [Lookup Table](/docs/search/lookup-tables) you can enrich it detailed SLO data (such as sloName, signalType, sliQueryType, sloDescription) using the `_view=sumologic_slo_output` snippet.
```
Comment (arpitjain305) [inline docs/observability/reliability-management-slo/create-slo.md:349]: @kimsauce the SLO Lookup Table is a different feature. It's basically an internal Sumo managed lookup table (i.,e the user does not have to create or manage this lookup table). It resides under a fixed path `sumo://content/slos` and contains the metadata of all the SLOs in the user's environment.

In the docs, we can describe that there are two ways to use this broadly:
1. `_view=sumologic_slo_output | lookup * from sumo://content/slos on sloId, sloVersion` - joins the results of the SLO precomputed data from _view = sumologic_slo_output with the metadata contained in the internal lookup table based on the joining key (sloId, sloVersion)
2. `cat sumo://content/slos` enlists the contents of the lookup table

Comment (arpitjain305) [reply docs/observability/reliability-management-slo/create-slo.md:349]: I think the current content here and below is around the `Open In` (drilldown action).

Comment (kimsauce) [reply docs/observability/reliability-management-slo/create-slo.md]: Thank you! I'll tag you in the revision to re-review

File: docs/observability/reliability-management-slo/create-slo.md (hunk 1 above)
Comment (arpitjain305) [reply docs/observability/reliability-management-slo/create-slo.md:373]: Sharing some screen shots:
Screenshot 2023-02-14 at 11 08 17 AM
">

Comment (arpitjain305) [reply docs/observability/reliability-management-slo/create-slo.md:373]: Screenshot 2023-02-14 at 11 08 17 AM

Comment (arpitjain305) [reply docs/observability/reliability-management-slo/create-slo.md:373]: Screenshot 2023-02-14 at 11 08 33 AM

Comment (arpitjain305) [reply docs/observability/reliability-management-slo/create-slo.md:373]: `SLO Summary from Lookup` - can be accessed on `us2Data`

Comment (kimsauce) [reply docs/observability/reliability-management-slo/create-slo.md:373]: thank you!

File: docs/observability/reliability-management-slo/create-slo.md [hunk 3]
```diff
@@ -345,3 +345,42 @@ _view=sumologic_slo_output sloId="<your-SLO-ID>"
...
+You can call a SLO Lookup Table to view all SLO metadata in your environment. These tables reside under a fixed path, `sumo://content/slos`. Data is managed and refreshed automatically on our end.
+
+There are two ways to use it:
+
+* To join the results of your SLO precomputed data from `_view = sumologic_slo_output` with your metadata contained in the internal lookup table based on the joining key (`sloId`, `sloVersion`):
+  ```sql
+  _view=sumologic_slo_output
+  | lookup * from sumo://content/slos on sloId, sloVersion
+  ```
+* To enlists the contents of the lookup table:
```
Comment (arpitjain305) [inline docs/observability/reliability-management-slo/create-slo.md:361]: minor: `To enlist`

**Pull request merged**